src/main/sensors/barometer.h

```diff
@@ -25,7 +25,8 @@ typedef enum {
     BARO_NONE = 1,
     BARO_BMP085 = 2,
     BARO_MS5611 = 3,
-    BARO_BMP280 = 4
+    BARO_BMP280 = 4,
+	BARO_LPS = 5
```

Comment:
Please use same indent. BF is only using spaces (1 tab is 4 spaces).

Reply:
Sure